src/kernel/kmain.zig

```diff
@@ -129,10 +130,12 @@ export fn kmain(boot_payload: arch.BootPayload) void {
 
         // Need to init the vfs after the ramdisk as we need the root node from the ramdisk filesystem
         vfs.setRoot(ramdisk_filesystem.root_node);
-
-        // Load all files here
     }
 
+    scheduler.init(&kernel_heap.allocator, &mem_profile) catch |e| {
+        panic_root.panic(@errorReturnTrace(), "Failed to initialise scheduler: {}\n", .{e});
+    };
+
```

Comment:
How come the scheduler was moved down, I don't think it makes a difference, am just curious.

Reply:
That's because the scheduler rt tests use a file from the ramdisk so need to be run after that is set up.